tests/resources/Util.robot

```diff
@@ -176,11 +176,8 @@ Cleanup Dangling VMs On Test Server
     \   ${state}=  Get State Of Drone Build  @{build}[1]
     \   Continue For Loop If  '${state}' == 'running'
     \   ${uuid}=  Run  govc vm.info -json\=true ${vm} | jq -r '.VirtualMachines[0].Config.Uuid'
-    \   Log To Console  Destroying dangling VM ${vm}
-    \   ${output}=  Run  govc vm.destroy ${vm}
-    \   ${output}=  Run  govc pool.destroy %{GOVC_RESOURCE_POOL}/${vm}
-    \   ${output}=  Run  govc datastore.rm ${vm}
-    \   ${output}=  Run  govc datastore.rm VIC/${uuid}
+    \   Log To Console  Destroying dangling VCH: ${vm}
+    \   ${rc}  ${output}=  Run Secret VIC Machine Delete Command  ${vm}
```

Comment:
Is the "Secret VIC Machine Delete Command" already defined somewhere else? Also do you need another space after `Run`?